logcat/gradle.properties

```diff
@@ -1,3 +1,25 @@
-POM_ARTIFACT_ID=logcat
-POM_NAME=Logcat
-POM_PACKAGING=aar
+kotlin.code.style=official
+#kotlin.mpp.enableGranularSourceSetsMetadata=true
+#kotlin.native.enableDependencyPropagation=false
+kotlin.js.generate.executable.default=false
+android.useAndroidX=true
+
+# Required to publish to Nexus (see https://github.com/gradle/gradle/issues/11308)
+systemProp.org.gradle.internal.publish.checksums.insecure=true
```

Comment:
I believe this issue is fixed and this property is no longer necessary.

Reply:
Looks like it was fixed in 6.0.1, so I can remove this property

Reply:
Removed in gradle.properties